src/components/NcSelect/NcSelect.vue

```diff
@@ -1086,12 +1086,16 @@ body {
 	}
 
 	&.vs--open .vs__dropdown-toggle {
-		border-color: var(--color-primary-element);
+		outline: 2px solid var(--color-main-text);
+		border-color: var(--color-main-background);
+		outline-offset: -2px;
```

Comment:
```suggestion
		outline: 2px solid var(--color-main-background);
		border-color: var(--color-main-text);
```

Should fix the issue with the "line" being shown

Reply:
Thanks!